pfcwd/main.py

```diff
@@ -127,6 +128,21 @@ def start(action, restoration_time, ports, detection_time):
         configdb.mod_entry("PFC_WD_TABLE", port, None)
         configdb.mod_entry("PFC_WD_TABLE", port, pfcwd_info)
 
+# Set WD poll interval
+@cli.command()
+@click.option('--poll_interval', '-p', type=click.IntRange(100, 3000))
```

Comment:
Argument instead of an option?
Options are usually optional, and it's not the case for poll interval,